src/elements/content-explorer/Content.js

```diff
@@ -22,9 +23,9 @@ import './Content.scss';
  * @param {Object} currentCollection the current collection
  * @return {boolean} empty or not
  */
-function isEmpty(view: View, currentCollection: Collection): boolean {
+function isEmpty(view: View, currentCollection: Collection, metadataFieldsToShow: MetadataFieldsToShow): boolean {
```

Comment:
you can add metadataFieldsToShow to the params definition above `* @param {MetadataFieldsToShow} metadataFieldsToShow`